test/srt/test_nightly_gsm8k_eval.py

```diff
@@ -52,7 +52,7 @@ def launch_server(self, model, is_fp8, is_tp2):
         if is_tp2:
             other_args.extend(["--tp", "2"])
         if "DeepSeek" in model:
-            other_args.append("--enable-mla")
+            other_args.extend(["--enable-mla", "--mem-frac", "0.85"])
```

Comment:
This is because after implementing CUDA Graph in Triton Runtime, the original mem-frac will cause OOM, so it is necessary to manually adjust mem-frac to bypass this issue. cc @merrymercy